setup.py

```diff
@@ -1,20 +1,34 @@
-from distutils.core import setup
-from distutils.extension import Extension
-from Cython.Distutils import build_ext
-from numpy import get_include
+from setuptools import find_packages, setup, Extension as _Extension
 
+
+class Extension(_Extension):
+    def __init__(self, *args, **kwargs):
+        super(Extension, self).__init__(*args, **kwargs)
+
+    @property
+    def include_dirs(self):
```

Comment:
Yeah I hacked [this class](https://github.com/python-git/python/blob/master/Lib/distutils/extension.py#L26), which is just a container for a bunch of attributes. I'm not sure exactly when it gets called, but it does seem to be after setuptools installs numpy.